data/locale/en-US.ini

```diff
@@ -139,10 +150,12 @@ NVENC.AQ.Temporal="Enable Temporal Adaptive Quantization"
 NVENC.AQ.Temporal.Description="Enable temporal adaptive quantization."
 NVENC.Other="Other Options"
 NVENC.Other.BFrames="Maximum B-Frames"
-NVENC.Other.BFrames.Description="Maximum number of B-Frames to insert into the encoded bitstream.\nActual number of B-Frames may be lower depending on content and lookahead settings."
+NVENC.Other.BFrames.Description="Maximum number of B-Frames to insert into the encoded bitstream.\nOnly Turing NVENC supports B-Frames for HEVC.\nActual number of B-Frames may be lower depending on content and lookahead settings."
 NVENC.Other.BFrameReferenceMode="B-Frame Reference Mode"
 NVENC.Other.BFrameReferenceMode.Each="Each B-Frame will be used for references"
 NVENC.Other.BFrameReferenceMode.Middle="Only (# of B-Frames)/2 will be used for references"
+#This is already explained in the drop-down list, but there is also a placeholder tooltip text. Is there a way to disable the tooltip here?
```

Comment:
Yes, that is possible.

Reply:
How to do this, exactly? I tried `""` and it still shows the same tooltip, I tried `" "` and it shows a tooltip with a whitespace character.

Reply:
I just have to remove the code in the ui handler for it.